scripts/run_integration_tests.sh

```diff
@@ -12,15 +12,15 @@ fi
 export TEST_BIN=$1
 LIST=$(bash ${PWD}/scripts/generate_test_list.sh $TEST_BIN)
 
-docker-compose up -d
-./scripts/wait_for_electrs.sh
 
 for TEST_NAME in $LIST
 do
     if [ ! -z $TEST_NAME ]
     then
+        docker-compose up -d
+        ./scripts/wait_for_electrs.sh
         RUST_MIN_STACK=104857600 cargo test -- $TEST_NAME --ignored --exact --nocapture
+        docker-compose down -v
```

Comment:
Why did you decide to start restarting docker between test runs? Is this used on CI or is it only for local development?

Reply:
That makes sense. Let's keep it this way then.